index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&family=Poppins:wght@100;200&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+</html>
+<body>
+  <header class="header">
+    <nav class="header_nav">
+      <div class="header_logo black"><a class="black" href="#">Welcome</a></div>
+      <div class="header_menu">
+        <img class="menu_icon" src="images/Union.png" alt="menu" />
+      </div>
+    </nav>
+  </header>
+  <section class="headline">
+    <h1 class="headline_title">
+      <span class="sub-headline_title_1">Hey there. I’m 
+        <br>Cansu</span>
+      <span class="sub-headline_title">I'm a software 
+        <br>developer</span>
+    </h1>
+    <p class="headline_content">
+      I can help you build a product , feature or website Look through some of
+      my work and experience! If you like what you see and have a project you
+      need coded, don’t hestiate to contact me. </p>
+    <div class="backimg"></div>
+    <div class="addline_social">
```

Comment:
- [ ] I appreciate the effort you put in implementing your design, but your design should stick as much as possible to the Figma design :100:. That means you should make sure to check if the attribute such as ```font-family```, ```color```, ```margin```, ```font-weight``` etc are the same in your design as in the Figma design. :smile: 
- [ ] Kindly make sure the ```font-family``` look the same in your design as given in the Figma design :health_worker: 

Your design:

![Screenshot from 2022-12-02 19-46-32](https://user-images.githubusercontent.com/80458199/205354117-9ca64bca-526d-493b-8524-338d83d41b61.png)

The Figma design

![Screenshot from 2022-12-02 19-47-13](https://user-images.githubusercontent.com/80458199/205354197-164420db-a93d-4e5d-b346-5af30c1c75d8.png)